Jenkinsfile_CNP

```diff
@@ -19,7 +19,8 @@ def branchesToSync = ['demo', 'ithc', 'perftest']
 
 // Variables to switch pipeline logic and wiring per type of build
 
-def definitionStoreDevelopPr = "PR-575"  // This doesn't change frequently, but when it does, only change this value.
+// TODO: remove TEMPORARY OVERRIDE to test GlobalSearch, reinstate: def definitionStoreDevelopPr = "PR-575"
+def definitionStoreDevelopPr = "PR-1279"  // This doesn't change frequently, but when it does, only change this value.
```

Comment:
This should be reveresed if final test-def version changes are merged to `master`.